rpcs3/Emu/RSX/rsx_methods.h

```diff
@@ -127,8 +127,8 @@ namespace rsx
 
 	struct rsx_state
 	{
-	protected:
-		std::array<u32, 0x10000 / 4> registers;
```

Comment:
This needn't not be an array. A single variable entry will suffice since the bound method is called immediately after the register set. The rest of the logic can remain the same.